### DISCUSSION TITLE
Get the Most Out of Universe!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#38793] [Announcements]
@damovisa, @anjuan, and @filmgirl have kicked us off for Day Two with another amazing [preshow](https://watch.githubuniverse.com/on-demand/63f45d9b-775f-49e8-8bdd-8254d87b17b6) 🍿 and a schedule packed with plenty of great sessions. But that isn't the only way to interact with Universe virtually, we have a special [community](https://github.com/community/events/discussions) just for Universe and we'd love to have you join us 💬 . 

There are plenty of exciting discussions to jump into... we don't want you to miss them: 
* @rmw's [What's Your Git Personality](https://github.com/community/events/discussions/48)? 
* [Git Out Of Here: How GitHub Does Detection ](https://github.com/community/events/discussions/27) from @calhall 
* [Share which sessions](https://github.com/community/events/discussions/58) you're looking forward to today with me, @queenofcorgis!
* [Code Search Q&A](https://github.com/community/events/discussions/59) with @colinwm 🔍 

React with us, share your thoughts with fellow attendees, and start discussions of your own. 

[JOIN THE US IN THE UNIVERSE COMMUNITY](https://github.com/community/events/discussions)!

**Comment by happy-spark** [comment]
Cool!

**Comment by Preetiraj3697** [comment]
Great 😀

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
